apps/src/templates/teacherDashboard/teacherSectionsRedux.js

```diff
@@ -376,8 +383,33 @@ export default function teacherSections(state=initialState, action) {
   if (action.type === SET_SECTIONS) {
     const sections = action.sections.map(section =>
       sectionFromServerSection(section));
+
+    let selectedSectionId = state.selectedSectionId;
+    // If we have only one section, autoselect it
+    if (Object.keys(action.sections).length === 1) {
+      selectedSectionId = action.sections[0].id.toString();
+    }
```

Comment:
This could also be done in the action itself (i.e. make it dispatch a SELECT_SECTION action). Not sure which is better, but erred towards minimizing thunk actions.